openapi/publicapi_spec.json

```diff
@@ -1440,7 +1440,7 @@
           "authorisation_mode" : {
             "type" : "string",
             "description" : "How the payment will be authorised. Payments created in `web` mode require the paying user to visit the `next_url` to complete the payment.",
-            "enum" : [ "web", "moto_api", "external" ]
+            "enum" : [ "web", "moto_api", "agreement", "external" ]
```

Comment:
Please remove `"agreement"` from this while recurring payments are still in private beta.

Reply:
👍